File: PapaKarloSwift/PapaKarloSwift/feature/profile/order/order_list/OrderItem.swift
//
//  OrderItem.swift
//  PapaKarloSwift
//
//  Created by Марк Шавловский on 12.03.2022.
//

import SwiftUI

struct OrderItem :Identifiable {
    let id:UUID
    let status:String
    let code:String
    let dateTime:String
}

File: feature/order/src/commonMain/kotlin/com/bunbeauty/order/ui/model/OrderItem.kt
package com.bunbeauty.order.ui.model

import androidx.compose.ui.graphics.Color
import com.bunbeauty.core.model.order.OrderStatus

data class OrderItem(
    val uuid: String,
    val status: OrderStatus,
    val statusName: String,
    val code: String,
    val dateTime: String,
    val background: Color,
)
